src/mmw/js/src/data_catalog/templates/resultDetails.html

```diff
@@ -0,0 +1,20 @@
+<div class="result">
+    <div class="result-detail-header">
+        <h2>
+            {{ title }}
+        </h2>
+        <button type="button" class="close" aria-label="Close">
+            <span aria-hidden="true">×</span>
+        </button>
+        <a href="#"><i class="fa fa-search-plus"></i> Zoom to extent</a>
+    </div>
+    <hr>
+    {% if author %}
+        <p>
+            <i class="fa fa-user"></i>{{ author }}
+        </p>
+    {% endif %}
+    <p>
+        {{ description }}
```

Comment:
I didn't implement this originally because the APIs handle dates in different ways. But after rereading the card, I'm going to take a shot at it.